server/db.go

```diff
@@ -230,6 +232,45 @@ func (c *Server) fetchIssuer(issuerID string) (*Issuer, error) {
 	return issuer, nil
 }
 
+func (c *Server) fetchIssuersByCohort(issuerType string, issuerCohort int) (*[]Issuer, error) {
+	if c.caches != nil {
+		if cached, found := c.caches["issuers"].Get(issuerType); found {
```

Comment:
does this caching have the right behavior? it seems like it doesn't depend on the cohort

in the following statement I would think the issuers for cohort 1 may be returned for cohort 2 since they would be written into the cache by the first call:
```
fetchIssuersByCohort('confirmation', 1)
fetchIssuersByCohort('confirmation', 2)
```